snippets/partial_payment.py

```diff
@@ -0,0 +1,88 @@
+"""A snippet that walks us through using a partial payment."""
+from xrpl.clients import JsonRpcClient
+from xrpl.models.amounts import IssuedCurrencyAmount
+from xrpl.models.requests import AccountLines
+from xrpl.models.transactions import Payment, PaymentFlag, TrustSet
+from xrpl.transaction import (
+    safe_sign_and_autofill_transaction,
+    send_reliable_submission,
+)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
+
+# creating wallets as prerequisite
```

Comment:
```suggestion
# Creating two wallets to send money between
```

Style nit: Comments should start with a capital letter (Ex. creating -> Creating)